authors/apps/articles/views.py

```diff
@@ -49,6 +51,32 @@ def post(self, request):
         return Response(serializer.data, status=status.HTTP_201_CREATED)
 
 
+class Tagview(GenericAPIView):
+    permission_classes = (IsAuthenticated,)
+
+    @staticmethod
+    def all_tags():
+        try:
+            articles = Article.objects.all()
+            articles_with_tags = list(filter(lambda x: x.tags, articles))
+            all_tags = []
+            if articles_with_tags:
+                all_tags_list = [x.tags for x in articles_with_tags]
+                for tag in all_tags_list:
+                    all_tags += tag
+                return set(all_tags)
+        except Article.DoesNotExist:
```

Comment:
@sebbss, instead of using a try catch, use a get_object_or_404 method .
Use of try catch blocks is highly discouraged.